go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/rancher/rancher
 
 go 1.22
 
+replace github.com/rancher/shepherd => github.com/caliskanugur/shepherd v0.0.0-20240507144611-b10827fbb1f1
```

Comment:
This will be switched back to rancher/shepherd, right?

Reply:
Yup, once this pull request is merged, automation team will proceed to update rancher/shepherd, which has a dependency on rancher/apis. Afterward, I'll ensure that this is updated with the appropriate rancher/shepherd tag.

Reply:
Just to keep track of this : https://github.com/rancher/rancher/issues/45564

Reply:
Cool, it sounds good to me.